src/main/java/com/codeborne/selenide/Condition.java

```diff
@@ -101,6 +105,26 @@ public boolean apply(WebElement element) {
    */
   public static final Condition disappear = hidden;
 
+  /**
+   * Checks that element is moving
+   *
+   * Element should implement {@link Locatable}
+   *
+   * <p>Sample: <code>$("#loginLink").shouldBe(moving);</code></p>
+   */
+  public static final Condition moving = new Condition("moving") {
+    @Override
+    public boolean apply(WebElement element) {
+      if (!(element instanceof Locatable)) {
+        throw new RuntimeException("Provided WebElement is not Locatable, cannot understand if it moving or not");
+      }
+      Point initialLocation = ((Locatable) element).getCoordinates().inViewPort();
+      Uninterruptibles.sleepUninterruptibly(200, MILLISECONDS);
```

Comment:
What behavior should be if animation took more that 200 milliseconds?
For example, some portion of test data loaded by 1 second, and it is single page react application, where page loaded before content.
So, what should we expect and do?

Reply:
We are just checking if location of an element has changes during 200ms, if it has - then it means that element is moving, otherwise it is not

Reply:
For example we can check when snackbar (https://www.w3schools.com/howto/howto_js_snackbar.asp) has fully appeared (moving animation is finished), and if we want to click a link on it is a must to do, otherwise sometimes click might not work

Reply:
Btw, 200ms it's magic number that can't be proof of good application work.
Maybe it's good to introduce configuration, something like `animationWait` or something like that to configure this sleep.
I think we need more ideas from @asolntsev and @BorisOsipov

Reply:
yes, is it magic number, but it has some assumptions:
smaller interval of webdriver calls introduces significant performance issues, especially if some any kind of waiter is which calls this condition.
Bigger waiting times makes test slower, so for it looks like 200ms is an optimal solution, but maybe there are some use cases where it should be configured, dunno

Reply:
If parameterize this condition I think it should be something like `$("#element").should(movesOver(200));`

Reply:
How about `moveAround`/`movingAround`? and what about not parametrised version with default value of 200?
```$("element").should(moveAround(200));
$("element").shouldBe(movingAround(200))```